joebvp/VPmeasure.py

```diff
@@ -25,16 +26,22 @@
 from linetools.spectra.io import readspec
 import numpy as np
 from astropy.constants import c
-
+try:
+    from importlib import reload
+except:
+    pass
```

Comment:
Is this necessary?

Reply:
I reload the cfg.py file is to clear the LSFs loaded for each spectral chunk fitted, and python 3 doesn't natively have a reload() function.  The `try except` business is because python 2 doesn't have the importlib package.  This allows some backward compatibility.